multicore_runtime/middleware.py

```diff
@@ -0,0 +1,185 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""WSGI middleware to wrap the dispatcher, and supporting functions."""
+
+import logging
+import os
+
+from werkzeug.wrappers import Request
+from werkzeug.wrappers import Response
+
+
+RESERVED_ENV_KEYS = {
```

Comment:
Looking at the code, I see the "value" in the dict is actually the default value to use if it's not present, please add a comment to document that here.